docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -2005,6 +2005,7 @@ Census
   * - First name
   * - Middle initial
   * - Last name
+  * - Year
```

Comment:
As I noted in the pseudopeople PR, let's check the order of the columns here. I think we'd want this at the beginning (maybe before even the unique IDs?)

Reply:
As I noted in the [pseudopeople PR](https://github.com/ihmeuw/pseudopeople/pull/241), year/tax_year is currently the _last_ column in the datasets in which it appears. Do we like that, or do we want to move it to the beginning?

Are the column orders always consistent between the concept model, pseudopeople docs, and the actual data? I'm not sure...

Reply:
@pletale As noted in the pseudopeople PR, let's list `year` / `tax_year` as the **last** column in the relevant datasets, to be consistent with what's already implemented.

Reply:
@zmbc @NathanielBlairStahn Just moved the year columns to the last column in the datasets (Census, WIC, taxes)!